### ISSUE TITLE
Decreased CSF glycine concentration

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7710]
**Preferred term label:**
Decreased CSF glycine concentration

**Synonyms**

**Definition (free text, please give PubMed ID)**
Abnormally decreased levels of glycine in cerebrospinal fluid.

**Parent term (use hpo.jax.org/app)**
Abnormal CSF glycine concentration HP:0500229

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
610992 
PMID: 17436247   2/2

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
